### ISSUE TITLE
NTR: Vitamin B2 (riboflavin) responsive (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8340]
New term request Vitamin B2 (riboflavin) responsive (elthat_130503094626):

**Comment by pnrobinson** [comment]
@MickeySegal
can you provide the context? Is it https://pubmed.ncbi.nlm.nih.gov/25251739/

**Comment by MickeySegal** [comment]
We also have it for https://omim.org/entry/615026, https://omim.org/entry/616839 and https://omim.org/entry/255100.  The literature uses the term "riboflavin-responsive" but a better finding name might be "Riboflavin reduces symptoms".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000373
- label: Ameliorated by riboflavin
- parents: [Ameliorated by (HP:0025254)](https://hpo.jax.org/app/browse/term/HP:0025254)
- pmid: 
- def: Applies to a sign or symptom that is improved or made more bearable by ingestion of riboflavin (Vitamin B2).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8340
- synonyms:
